src/components/__tests__/__snapshots__/RefinementList.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`allows search bar classes override when it's searchable 1`] = `
 
-<div class="ais-RefinementList ais-RefinementList--noRefinement">
+<div class="ais-RefinementList">
```

Comment:
this is because the test used to be wrong (didn't pass `canRefine` in defaultState)